5/이상한문자만들기.js

```diff
@@ -0,0 +1,17 @@
+function solution(s) {
+  const words = s.split(" ");
+  const results = [];
+
+  for (let word of words) {
+    let s = "";
+
+    for (let i in word) {
+      if (i % 2 === 0) s += word[i].toUpperCase();
+      else s += word[i].toLowerCase();
+    }
+
+    results.push(s);
+  }
+
+  return results.join(" ");
```

Comment:
```suggestion
function solution(s) {
    return s.split(' ')
        .map(word => word.split('')
        .map((caseStr, index) => index % 2 ? caseStr.toLowerCase() : caseStr.toUpperCase())
        .join('')).join(' ');
}
```
이 문제 또한 map함수를 두번 사용하면 훨씬 간결하게 코드를 짤 수 있어요. 
반복문보다 map 함수를 쓸 수 있도록 한 번 연습해보세요.